README.md

```diff
@@ -90,6 +90,10 @@ For example, if you want to get a summary report for the last week, you should r
 
 `$ php /usr/local/share/dmarc-srg/utils/summary_report.php domain=example.com period=lastweek`
 
+If you receive an error message suggesting register_argc_argv is disabled, you can enable it like this:
+
+`$ php -d register_argc_argv=1 /usr/local/share/dmarc-srg/utils/summary_report.php domain=example.com period=lastweek`
```

Comment:
```suggestion
`$ php -d register_argc_argv=1 -f /usr/local/share/dmarc-srg/utils/summary_report.php domain=example.com period=lastweek`
```

Reply:
Using the `-f` flag may be technically correct and may be good practice. I didn't think to add it in my setup and it works well. It doesn't seem to be required [from the PHP manual for running a file](https://github.com/liuch/dmarc-srg/pull/138#pullrequestreview-2369423543):

```
Usage: php [options] [-f] <file> [--] [args...]
```

The `<file>` is not inside the optional `[-f]` parameter. It's less compact, but perhaps should be included to assist anyone who might get confused adding multiple options. It may reduce the chance of someone causing errors if they add other option parameters.